exporter/newrelicexporter/transformer.go

```diff
@@ -33,8 +33,8 @@ const (
 	descriptionAttrKey        = "description"
 	collectorNameKey          = "collector.name"
 	collectorVersionKey       = "collector.version"
-	instrumentationNameKey    = "instrumentation.name"
-	instrumentationVersionKey = "instrumentation.version"
+	instrumentationNameKey    = "otel.library.name"
+	instrumentationVersionKey = "otel.library.version"
```

Comment:
These are defined in `conventions` package from the core Collector repo. It is probably best to just import that.